### PR TITLE
Fix trycatch hanging

### PR DESCRIPTION
**Pull request by alexvpickering**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-1392

https://biomage.atlassian.net/browse/BIOMAGE-1556

#### Link to staging deployment URL 
https://ui-alex-ui569-pipe190.scp-staging.biomage.net/

#### Links to any Pull Requests related to this

https://github.com/biomage-ltd/ui/pull/569

###  --> RESTORE STAGING RAM BEFORE MERGE <--

**Test data**: https://drive.google.com/drive/u/1/folders/1ioY1k6Or_461NtbH60npiCJfy8DPgKs6
**NOTE**: hangs in productions, not in staging

Cause of issue: https://github.com/aryoda/tryCatchLog/issues/68

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [x] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [x] Deployed to staging
- [ ] Passed integration tests 

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -231,7 +231,7 @@ jobs:
             KUBERNETES_ENV="staging"
             REPLICA_COUNT="0"
             IMAGE_GLOB="${IMAGE_TAG/$GITHUB_SHA/*}"
-            MEMORY_REQUEST="4Gi"
+            MEMORY_REQUEST="16Gi"
```
Comment (ivababukova) [inline .github/workflows/ci.yaml:234]: do you still need this change? I assume it was added just for testing?

File: pipeline-runner/DESCRIPTION [hunk 2]
```diff
@@ -18,7 +18,8 @@ Suggests:
     testthat (>= 3.0.0)
 Imports:
     data.table,
-    covr
+    covr,
+    batchelor
```
Comment (ivababukova) [inline pipeline-runner/DESCRIPTION:22]: are you using this?

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (alexvpickering) [reply .github/workflows/ci.yaml:234]: Yes just for testing

File: pipeline-runner/DESCRIPTION (hunk 2 above)
Comment (alexvpickering) [reply pipeline-runner/DESCRIPTION:22]: yes. This is an addtional package needed for fastmnn

**Review by ivababukova — APPROVED** [review]

**Pull request merged**